CODEOWNERS

```diff
@@ -1,53 +1,53 @@
 # NOTE: Disabled temporarily because it's too noisy on pushes.
 # Where component owners are known, add them here.
 
-#tensorflow/core/platform/windows/* @mrry
-#tensorflow/java/* @asimshankar
-#tensorflow/tensorboard/* @jart @dandelionmane
-#tensorflow/tools/docs/* @markdaoust
+# /tensorflow/core/platform/windows/ @mrry
```

Comment:
Don't we also need to remove the `#` in the beginning of these lines?

Reply:
The file is currently disabled, which I'm not aiming to reverse. I'm just fixing up the file syntax in case we end up re-enabling them in the future.

I did the same for the models repo, which has a CODEOWNERS file that's actively being used: https://github.com/tensorflow/models/pull/3016